test/rekt/resources/containersource/containersource_test.go

```diff
@@ -126,3 +129,68 @@ func Example_full() {
 	//         - name: POD_NAMESPACE
 	//           value: bar
 }
+
+func Example_sink() {
+	ctx := testlog.NewContext()
+	images := map[string]string{}
+	cfg := map[string]interface{}{
+		"name":      "foo",
+		"namespace": "bar",
+		"args":      "--period=1",
+		"ceOverrides": map[string]interface{}{
+			"extensions": map[string]string{
+				"ext1": "val1",
+				"ext2": "val2",
+			},
+		},
+	}
+
+	sinkRef := &duckv1.Destination{
+		Ref: &duckv1.KReference{
+			Kind: "AKind",
+			// Namespace:  "sinknamespace",
```

Comment:
To make this work we need to update `test/rekt/resources/containersource/containersource.yaml` from
```
      kind: {{ .sink.ref.kind }}
      namespace: {{ .namespace }}
```
to:
```
      kind: {{ .sink.ref.kind }}
      namespace: {{ .sink.ref.namespace }}
```